elixir/apps/domain/priv/repo/migrations/20230801154306_create_policies.exs

```diff
@@ -38,7 +38,13 @@ defmodule Domain.Repo.Migrations.CreatePolicies do
       timestamps(type: :utc_datetime_usec)
     end
 
-    create(index(:policies, [:account_id, :name], unique: true))
-    create(index(:policies, [:account_id, :resource_id, :actor_group_id], unique: true))
+    create(index(:policies, [:account_id, :name], unique: true, where: "deleted_at IS NULL"))
```

Comment:
👍

Reply:
This actually reminded me that I need to pull this out in to it's own migration so we don't have to completely reset the staging DB.

Reply:
Moved this change to it's own migration and reverted this file to it's previous state.